salt/utils/__init__.py

```diff
@@ -2563,7 +2562,11 @@ def is_dictlist(data):
     return False
 
 
-def repack_dictlist(data):
+def repack_dictlist(data,
```

Comment:
I know what the problem is, here. Firstly, I did not code in a case where ``strict`` is actually used, and secondly it should default to ``False``, not ``True``, to preserve expected behavior with existing code.